elasticsearchEC2.tf

```diff
@@ -0,0 +1,85 @@
+module "network" {
+  source             = "./vpc"
+  public_name               = "my_vpc"
```

Comment:
This refers to the public subnet name in the VPC which should be Kabana-Public-A

Reply:
I'm going to change that. Thanks